cdap-ui/app/features/hydrator/toppanel.less

```diff
@@ -58,34 +59,56 @@ body.theme-cdap {
               .opacity(1);
             }
           }
+          > div {
+            &:before {
+              content: none;
+              color: gray;
+            }
+            &:first-child {
+              font-size: 16px;
+              &.placeholder {
+                &:before {
+                  content: "[Pipeline Name]";
+                }
+              }
+            }
+            &:last-child {
+              font-size: 12px;
+              &.placeholder {
+                &:before {
+                  content: "Enter a description for your pipeline";
```

Comment:
Could be a minor thing but if we have ```[``` & ```]``` around Pipeline Name placeholder should we have one around this too?